presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve dictionary writer performance.
+* Add BigQuery Connector Plugin.
+* Added support for memozing dead nodes. This can be enabled by setting the ``internal-communication.memoize-dead-nodes-enabled`` configuration property.
```

Comment:
```suggestion
* Added support for memoizing dead nodes. This can be enabled by setting the ``internal-communication.memoize-dead-nodes-enabled`` configuration property.
```

Also, it might not be clear to a Presto operator what "memoizing dead nodes" means.  Need to come up with some better clearer phrasing.

Reply:
@NikhilCollooru can you please add a clearer description of what this property does.

Reply:
How about the following :

Add support for a resilient cache affinity logic by memoizing the dead nodes. This will help in having a high data cache hit rate even if some nodes go down for a limited time. This can be enabled by setting the ``internal-communication.memoize-dead-nodes-enabled`` configuration property.

Reply:
@NikhilCollooru 's suggestion while clarifies this, it is too long - 

```
Add support for improving cache affinity hit rate by preventing repartitioning when nodes go down for a limited time. This can be enabled by setting the `internal-communication.memoize-dead-notes-enabled` configuration property
```